framework/src/modules/chain/dpos/delegates_list.js

```diff
@@ -88,11 +88,30 @@ class DelegatesList extends EventEmitter {
 	}
 
 	async deleteDelegateListUntilRound(round) {
+		this.deleteDelegateListsFromCache(r => r < round);
```

Comment:
Can you pick a more meaningful name for `r`? :)
Something like `roundFromCache`?